src/blockchain/contracts/lido.py

```diff
@@ -17,3 +17,11 @@ def get_depositable_ether(self, block_identifier: BlockIdentifier = 'latest') ->
         response = self.functions.getDepositableEther().call(block_identifier=block_identifier)
         logger.info({'msg': 'Call `getDepositableEther()`.', 'value': response, 'block_identifier': repr(block_identifier)})
         return response
+
+    def get_buffered_ether(self, block_identifier: BlockIdentifier = 'latest') -> Wei:
+        """
+        Returns buffered ether.
```

Comment:
Usually I'm trying to copy description from contracts if there is something useful

Reply:
Copied from lido contract